app/src/main/java/com/sopt/dive/presentation/mypage/MyPageScreen.kt

```diff
@@ -78,7 +111,7 @@ private fun MyPageScreen(
         ) {
             Image(
                 painter = painterResource(id = R.drawable.profile),
-                contentDescription = "${userName} 프로필 이미지",
+                contentDescription = "$userName 프로필 이미지",
```

Comment:
```suggestion
                contentDescription = "${userName} 프로필 이미지",
```
요렇게 원래대로 쓰는게 가독성 더 좋을 것 같튼디 ?!